### PR TITLE
Alert button

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
code in documentation will be added soon - avatar/badge commits added by mistake, please ignore

image

image

image

### CODE REVIEW

File: styles/05-objects/badge.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  transform: translate(0%, -20%)
+}
+button .badge-sm{
+  top:-5px;
+}
+button .badge-lg{
+  top:-15px;
+}
+.badge-sm{
+  font-size: clamp(0.75rem, 1rem , 1.25rem);
```
Comment (SHIKHASINGH1506) [inline styles/05-objects/badge.css:31]: Nice implementation of clamp

**Review by SHIKHASINGH1506 — APPROVED** [review]
Things are on place for me. Good to merge

File: styles/01-settings/brand.css [hunk 2]
```diff
@@ -2,30 +2,39 @@
...
+  color: #ffffff;
 }
 .outline-secondary {
-  border: 1px solid var(--color-secondary);
+  border: 2px solid var(--color-secondary);
 } 
 
 .bg-accent {
   background-color: var(--color-accent);
+  color: #ffffff;
```
Comment (devdeadviz) [inline styles/01-settings/brand.css:27]: Use variables for color property also.

File: styles/99-trumps/status.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.status-available-bg {
+  background-color: #5686c5;
+  color: #202020;
+}
+
+.status-inactive-bg {
+  background-color: #615f5f;
+  color: #000000;
```
Comment (devdeadviz) [inline styles/99-trumps/status.css:132]: Try to use variables for colors, it provides you more flexibility.

**Review by devdeadviz — COMMENTED** [review]
Just one suggestion, use variables for colors, your code is clean. Good work!

File: styles/01-settings/brand.css (hunk 2 above)
Comment (HarshnaNHaswani) [reply styles/01-settings/brand.css:27]: Will note that down for code refactor. Thanks!

File: styles/05-objects/badge.css (hunk 1 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/badge.css:31]: Thank you!

File: styles/01-settings/brand.css (hunk 2 above)
Comment (HarshnaNHaswani) [reply styles/01-settings/brand.css:27]: Done!

Comment (HarshnaNHaswani) [reply styles/01-settings/brand.css:27]: Thank you!

File: styles/99-trumps/status.css (hunk 3 above)
Comment (HarshnaNHaswani) [reply styles/99-trumps/status.css:132]: Thanks! I made the changes

**Pull request merged**